src/components/common/TmBalance.vue

```diff
@@ -457,6 +479,55 @@ export default {
 }
 </script>
 <style scoped>
+select {
+  background: var(--input-bg);
+  color: var(--txt, #333);
+  border: none;
+}
+
+select option {
+  background: var(--app-bg);
+  color: var(--txt);
+  font-family: var(--sans);
+}
+
+.currency-flag {
+  width: 15px;
+  margin-right: 3px;
+}
+
+.currency-selector {
+  display: flex;
+  align-items: center;
+}
+
+.total-fiat-value {
+  min-width: 30px;
+  margin-top: 0.25rem;
+}
+
+.user-box {
```

Comment:
what makes this a good class name?

Reply:
it is where I copied most of the CSS from xD

OK, I change it 😅